README.md

```diff
@@ -109,7 +109,7 @@ needed modifications to the code for the module. Because of that, we ask users
 with general support, "how-to", or "why isn't this working" questions to try one
 of the other support channels that are available.
 
-Your first-stop-shop for support for webpack-dev-server should by the excellent
+Your first-stop-shop for support for webpack-dev-server should start by the excellent
```

Comment:
should "be" the excellent
instead of
should "by" the excellent

But "start by" sounds good too!